tests/helpers/common/Chocolatey/Get-ChocolateySource.ps1

```diff
@@ -0,0 +1,14 @@
+﻿function Get-ChocolateySource {
+    [CmdletBinding()]
+    param(
+        [Parameter()]
+        [string]$Name = "*",
+
+        [Parameter()]
+        [switch]$All

```

Comment:
Am I missing where this is being used?

Reply:
Nope... I copied it from the Enable/Disable functions... But it doesn't make sense on a `Get` function. `Get-ChildItem` gets me all the child items (except hidden, but that's a different story, and that uses force anyways). I'll take it off of here.

Reply:
I took this out since it's not used.